### ISSUE TITLE
Problem installing on Android

### ISSUE THREAD

**Issue by neomib** [Kishanjvaghela/react-native-cardview#30]
Hello
When I tried building the app for Android I got this error:
```
* What went wrong:
A problem occurred evaluating project ':app'.
> Could not get unknown property 'compile' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```
Any idea?

**Comment by b02505048** [comment]
+1

**Comment by Kishanjvaghela** [comment]
Published to v2.0.3 
https://www.npmjs.com/package/react-native-cardview
Feel free to reopen the issue if not working.
